.github/workflows/lint.yml

```diff
@@ -0,0 +1,30 @@
+name: Lint
```

Comment:
Change the name, I don't like `lint`

Reply:
What name to you suggest?

Reply:
I like the name `Ballad-Test`, as it seems proper as we're using your module that is SUPER helpful.